doc/botbuilder/interfaces/botbuilder.activity.md

```diff
@@ -574,9 +574,9 @@ ___
 
 ### «Optional» topicName
```

Comment:
what is the purpose of this member? we've been creating our own flags to track the conversation topic...is that what this is intended for?